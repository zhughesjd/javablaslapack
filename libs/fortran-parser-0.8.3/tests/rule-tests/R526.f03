! Testing data-stmt-object (R526), data-implied-do (R527), data-i-do-object (R528),
! and data-i-do-variable (R528)
data truth, beauty, ugly /3,14,15/
data a, (z%d, (B(i,j) , j=1,i), i=1,100,foo()) /3*1.0/ 
data myname /person(21,'john')/
data a%b/jj/
end
