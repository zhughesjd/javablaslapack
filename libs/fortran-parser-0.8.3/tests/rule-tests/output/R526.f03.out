1101:begin
526:list-begin
613 truth false false
612 1
603 false
601
613 beauty false false
612 1
603 false
601
613 ugly false false
612 1
603 false
601
526:list 3
530:list-begin
406 3 null
406 14 null
406 15 null
530:list 3
526:list-begin
613 foo false false
612 1
603 false
601
526:list 1
530:list-begin
406 3 null
530:list 1
1103
1101 false false false
